### PR TITLE
New feature nav

### PR DESCRIPTION
**Pull request by Jam52**
First story for board game market place:

**As a** board game enthusiast
**I want** to see a list of boardgames ordered by rating
**So that** I can scout out potential new purchases

Acceptance criteria:
- Board games list has a maximum size of 30
- Board games must be ordered by rating
- A game listing should consist of a name, picture and rating
- Design should be responsive
- Data should be dummy for now

Feedback on testing, file structure and code practices would be great.

Thank you!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,58 @@
...
+    if(this.state.categoryOptions.length > 0) {
+      categories = this.state.categoryOptions.map(category => {
+        return <option value={category.id} key={category.id}>
+          {category.name}
+        </option>
+      })
+    }
+
+    let gameCards = null;
+    if(this.state.loading === true) {
```
Comment (Nicholassully) [inline src/App.js]: This could be just `this.state.loading` as it will either be true or false maybe?

File: src/App.js [hunk 2]
```diff
@@ -1,26 +1,58 @@
...
+      gameCards = <Spinner/>
+    } else if (this.state.games.length > 0) {
+      gameCards = <GameCardList games={this.state.games} />
+    }
+
+    return (
+      <BrowserRouter>
+        <div className={classes.App}>
+          <select className={classes.Dropdown} onChange={event => this.categoryChangeHandler(event)}>
```
Comment (Nicholassully) [inline src/App.js]: I would advise not using `Dropdown` and making `dropdown` the convention in css is not to have capitals

File: src/Components/GameCardList/GameCard/GameCard.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        data-test="half-star"
+        src={halfStarIcon}
+        alt="half star"
+        key={index}
+      />
+    );
+  });
+
+  return (
+    <div data-test="component-game-card" className={classes.Container}>
```
Comment (Nicholassully) [inline src/Components/GameCardList/GameCard/GameCard.js]: Same as above will not write it for all of them :) in regards to capital on the css name `Container`

**Review by Nicholassully — COMMENTED** [review]
Just some minor things around convention rather than anything else but it's looking good from what I can see.

File: src/App.js (hunk 2 above)
Comment (Jam52) [reply src/App.js]: Thanks!! Let me know if you see any other convention things you notice!

**Pull request merged**